.gitignore

```diff
@@ -11,3 +11,6 @@ config/
 
 # pyenv
 .python-version
+
+# Juno artifacts (created when following README.md and running juno locally)
```

Comment:
it's default folder for storing db files

Reply:
let me update the comment, thank you